vertical-pod-autoscaler/README.md

```diff
@@ -54,100 +54,9 @@ as described in the next section.
 Please also have a look there if you want to understand the rationale behind
```

Comment:
Can you update the previous section too? 
> To receive recommendations please migrate to `autoscaling.k8s.io/v1beta2` API,
as described in the [migration doc]().

Reply:
done